tikv/kv.go

```diff
@@ -127,6 +127,9 @@ type KVStore struct {
 
 	replicaReadSeed uint32 // this is used to load balance followers / learners when replica read is enabled
 
+	// StartTS -> RURuntimeStats, stores the RU runtime stats for certain transaction.
+	ruRuntimeStatsMap sync.Map
```

Comment:
Seems the keys in this map is never deleted or clean, will this cause memory usage problem?

Reply:
I added a goroutine to do the clean-up work, PTAL again.